hazelcast-spring-tests/src/test/java/com/hazelcast/spring/TestBeansApplicationContext.java

```diff
@@ -80,6 +80,9 @@ public void testApplicationContext() {
         HazelcastClientProxy client = (HazelcastClientProxy) HazelcastClient.getAllHazelcastClients().iterator().next();
         assertNull(client.getClientConfig().getManagedContext());
 
+        int batchSize = client.getClientConfig().getQueryCacheConfigs().get("").get("cache1").getBatchSize();
```

Comment:
unrelated to this PR: I would expect the query cache config to be located like this:
```
client.getClientConfig().getQueryCacheConfigs().get("map1").get("cache1")
```

(instead of `get("")`). See also `ClientConfig#addQueryCacheConfig` method impl. This looks like an issue that was already there with spring config but went unnoticed.